exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillReduceAggregatesRule.java

```diff
@@ -218,7 +218,8 @@ private void reduceAggs(
         RelOptUtil.createProject(
             newAggRel,
             projList,
-            oldAggRel.getRowType().getFieldNames());
+            oldAggRel.getRowType().getFieldNames(),
+            DrillRelFactories.LOGICAL_BUILDER);
```

Comment:
Could you explain why we are using DrillRelFactories.LOGICAL_BUILDER but not relBuilderFactory that was used in line 211? And could you point me to this 4 param createProject method with Factory as the last param?

Reply:
In the second commit it was fixed and used relBuilderFactory to create builder and project.